DNN Platform/Library/Entities/Portals/IPortalStylesController.cs

```diff
@@ -0,0 +1,44 @@
+﻿#region Copyright
+
+// 
+// DotNetNuke® - http://www.dotnetnuke.com
```

Comment:
For new files, can we make the copyright match the new standard for .NET Foundation?

// Copyright (c) .NET Foundation. All rights reserved.  
// Licensed under the MIT License. See LICENSE file in the project root for full license information.